cmake/OpenCVFindIPPIW.cmake

```diff
@@ -108,21 +108,14 @@ macro(ippiw_setup PATH BUILD)
           message(STATUS "found Intel IPP Integration Wrappers binaries: ${IW_VERSION_MAJOR}.${IW_VERSION_MINOR}.${IW_VERSION_UPDATE}")
           message(STATUS "at: ${IPP_IW_PATH}")
 
-          add_library(ippiw STATIC IMPORTED)
-          set_target_properties(ippiw PROPERTIES
+          add_library(ipp_iw STATIC IMPORTED)
+          set_target_properties(ipp_iw PROPERTIES
             IMPORTED_LINK_INTERFACE_LIBRARIES ""
             IMPORTED_LOCATION "${FILE}"
           )
-          if (NOT BUILD_SHARED_LIBS)
```

Comment:
Instead of complete dropping this code, please add `OPENCV_INSTALL_EXTERNAL_DEPENDENCIES` option:
For example as list, including "all" (`ipp` in this case) - for better configuration. See [here](https://github.com/opencv/opencv/commit/7465125f67fa8c535477193e0730867a7a88136a#diff-a8bb897ccface5bd5e7eff5548b73f77).